### PR TITLE
pipeline page should not 500 if records not populated in DB

### PR DESCRIPTION
**Pull request by gidsg**
### Context
Previously the pipeline page would 500 on `get_register_definition` if any of the non-backlog registers were not populated in the database

### Changes proposed in this pull request
Pipeline page should only show registers where data is populated PLUS registers in backlog

### Guidance to review
Pipeline page should still load (e.g. if Charity has not been populated)

### CODE REVIEW

File: app/views/registers/index.html.haml [hunk 1]
```diff
@@ -87,7 +87,7 @@
                 = sort_link(@search, :register_authority, "Managed by", default_order: :asc, hide_indicator: true)
               %th{role: "columnheader", scope: "col"}
           %tbody
-            - @registers.each_with_index do |register, index|
+            - @registers.select(&:available?).each_with_index do |register, index|
```
Comment (danmitchell-) [inline app/views/registers/index.html.haml]: Can we move the method into the controller?

`@registers = Register.something.select(&:available?)`

File: app/controllers/registers_controller.rb [hunk 2]
```diff
@@ -6,15 +6,11 @@ class RegistersController < ApplicationController
...
-    @registers = if params[:phase] == 'ready to use'
-                   @search.result.where(register_phase: 'Beta').sort_by_phase_name_asc.by_name
-                 elsif params[:phase] == 'in progress'
-                   @search.result.where.not(register_phase: 'Beta').sort_by_phase_name_asc.by_name
-                 else
-                   @search.result.sort_by_phase_name_asc.by_name
-                 end
+    @search = Register.available.ransack(params[:q])
+
+    @registers = @search.result.sort_by_phase_name_asc.by_name
```
Comment (gidsg) [inline app/controllers/registers_controller.rb]: can we make these variants a case statement?

File: app/views/registers/index.html.haml [hunk 3]
```diff
@@ -48,7 +48,7 @@
   - else
     .grid-row
       .column-full
-        %h1 Registers pipeline
+        %h1.heading-xlarge Registers pipeline
```
Comment (gidsg) [inline app/views/registers/index.html.haml]: did you intend this change to go in there?

Comment (danmitchell-) [reply app/views/registers/index.html.haml]: Yeah. #cleanasyougo

**Pull request merged**